onnxruntime/core/mlas/lib/qgemm.cpp

```diff
@@ -1508,8 +2038,14 @@ Return Value:
     }
 
     GemmU8X8Operation(&WorkBlock);
-#else
+#elif defined(MLAS_SSE2_INTRINSICS)
     MlasGemmU8X8Operation<MLAS_GEMM_U8X8_KERNEL_SSE>(&WorkBlock);
```

Comment:
What about MLAS_VSX_INTRINSICS for power? Do we support it for quantized MatMul?

Reply:
That could come later: none of the quantization support has been implemented for VSX yet.